spec/main-spec.js

```diff
@@ -52,6 +52,14 @@ describe('SSH2', function() {
       privateKey: PRIVATE_KEY_PATH,
     })
   }
+  async function connectWithInlinePrivateKey(port, client) {
```

Comment:
Would be wonderful to inline this because we are not using it in any other place